v3/src/pages/docs/core-concepts/activity.md

```diff
@@ -117,6 +113,33 @@ The following is an example of annotating an activity with metadata:
     Description = "A simple activity that writes \"Hello World!\" to the console."
 )]
 ```
+---
+## Outcomes
+
+Custom Activity outcome Nodes can be defined as follows by Annotatting the the Metadata with the "FlowNode" attribute as follows:
+
+```clike
+[Activity(
+       "Demo", "Demo", "Simple activity ")]
+    [FlowNode("Pass", "Fail")]
+```
+This produces 2 outcomes of the Activity a in this case a Pass or Fail, this can be implemented as follows 
```

Comment:
"This produces two outcomes of the activity. In this case, a "Pass" or "Fail". To produce an outcome, use the `CompleteActivityAsync` method, as follows:"